source/features/remove-projects-tab.tsx

```diff
@@ -7,7 +7,7 @@ import * as pageDetect from 'github-url-detection';
 import features from '.';
 import {observeOneMutation} from '../helpers/simplified-element-observer';
 
-function getProjectsTab() {
+async function getProjectsTab(): Promise<HTMLElement | undefined> {
```

Comment:
I’m not particularly fond of the rule that enforces this (“promise functions must be async). If it’s the only case where this happens, fine, but if it requires a lot of similar unnecessary changes let’s remove it

Reply:
Ok, good as is.